doc/examples/registration/plot_stitching.py

```diff
@@ -0,0 +1,174 @@
+# matched to the reference points.
+
+img0 = img_list[0]
+coords0 = corner_list[0]
+matching_corners = [match_locations(img0, img1, coords0, coords1, min_dist)
+                    for img1, coords1 in zip(img_list, corner_list)]
+
+############################################################################
+# Once all the points are registered to the reference points, robust
+# relative affine transformations can be estimated using the RANSAC method.
+src = np.array(coords0)
+trfm_list = [measure.ransac((dst, src),
+                            transform.EuclideanTransform, min_samples=3,
+                            residual_threshold=2, max_trials=100)[0].params
+             for dst in matching_corners]
+
+fig, ax_list = plt.subplots(6, 2, figsize=(6, 9), sharex=True, sharey=True)
+for idx, (im, trfm, (ax0, ax1)) in enumerate(zip(img_list, trfm_list,
+                                                 ax_list)):
+    ax0.imshow(im, cmap="gray", vmin=0, vmax=1)
+    ax1.imshow(transform.warp(im, trfm), cmap="gray", vmin=0, vmax=1)
+
+    if idx == 0:
+        ax0.set_title(f"Tilted images")
+        ax0.set_ylabel(f"Reference Image\n(PSNR={psnr_ref:.2f})")
+        ax1.set_title(f"Registered images")
+
+    ax0.set(xticklabels=[], yticklabels=[], xticks=[], yticks=[])
+    ax1.set_axis_off()
+
+fig.tight_layout()
+
+############################################################################
+# Image assembling
+# ----------------
+#
+# A composite image can be obtained using the positions of the
+# registered images relative to the reference one. To do so, we define
+# a global domain around the reference image and position the other
+# images in this domain.
+#
+# A global transformation is defined to move the reference image in the
+# global domain image via a simple translation:
+margin = 50
+height, width = img_list[0].shape
+out_shape = height + 2 * margin, width + 2 * margin
+glob_trfm = np.eye(3)
+glob_trfm[:2, 2] = -margin, -margin
+
+############################################################################
+# Finally, the relative position of the other images in the global
+# domain are obtained by composing the global transformation with the
+# relative transformations:
+global_img_list = [transform.warp(img, trfm.dot(glob_trfm),
+                                  output_shape=out_shape,
+                                  mode="constant", cval=np.nan)
+                   for img, trfm in zip(img_list, trfm_list)]
+
+all_nan_mask = np.all([np.isnan(img) for img in global_img_list], axis=0)
+global_img_list[0][all_nan_mask] = 1.
+
+composit_img = np.nanmean(global_img_list, 0)
+psnr_composit = metrics.peak_signal_noise_ratio(
+    ref_img,
+    composit_img[margin:margin + height,
+                 margin:margin + width])
+
+fig, ax = plt.subplots(1, 1)
+
+ax.imshow(composit_img, cmap="gray", vmin=0, vmax=1)
+ax.set_axis_off()
+ax.set_title(f"Reconstructed image (PSNR={psnr_composit:.2f})")
```

Comment:
Why not just spell out composite fully in these variable names? (composit is only saving 1 character)

```suggestion
composite_img = np.nanmean(global_img_list, 0)
psnr_composite = metrics.peak_signal_noise_ratio(
    ref_img,
    composite_img[margin:margin + height,
                  margin:margin + width])

fig, ax = plt.subplots(1, 1)

ax.imshow(composite_img, cmap="gray", vmin=0, vmax=1)
ax.set_axis_off()
ax.set_title(f"Reconstructed image (PSNR={psnr_composite:.2f})")
```

Reply:
Thank you @grlee77 for catching this, it is a misspelling :confounded: